specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-09-01/costmanagement.json

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
In the future, please copy an unmodified version of the most recent previous version file into this PR as the first commit, and then update to new API version/changes in your second commit. That allows diff in the PR, and saves review time.

Reply:
Sure